core/src/main/java/io/lionweb/lioncore/java/self/LionCore.java

```diff
@@ -176,6 +187,14 @@ public static Language getInstance() {
       annotation.addFeature(
           Reference.createMultiple("implements", iface, "-id-Annotation-implements"));
 
+      structuredDataType.setExtendedConcept(dataType);
+      structuredDataType.addFeature(
```

Comment:
Is this _required_ by default?

Reply:
Do you mean `fields`? No it is multiple and multiple is not required by default, it is optional. So the multiplicity is 0..*

Reply:
in the spec M3 it's required

Reply:
Oh, I see. I have changed it accordingly